nidn/fdtd_integration/compute_spectrum_fdtd.py

```diff
@@ -33,11 +34,11 @@ def compute_spectrum_fdtd(permittivity, cfg: DotMap):
 
     if (len(cfg.PER_LAYER_THICKNESS) == 1) and (cfg.N_layers > 1):
         cfg.PER_LAYER_THICKNESS = cfg.PER_LAYER_THICKNESS * cfg.N_layers
-    thicknesses = torch.tensor(cfg.PER_LAYER_THICKNESS)
+    cfg.thicknesses = torch.tensor(cfg.PER_LAYER_THICKNESS)
```

Comment:
why not just overwrite the old variable? ( I mean cfg.PER_LAYER_THICKNESS)

Reply:
Sure, can do that